assets/js/romo/indicator_text_input.js

```diff
@@ -75,12 +77,18 @@ RomoIndicatorTextInput.prototype.doPlaceIndicatorElem = function() {
       this._hide(this.indicatorElem);
     }
 
-    var indicatorWidthPx = this.elem.data('romo-indicator-text-input-indicator-width-px') || this.defaultIndicatorWidthPx;
-    // left-side spacing
+    var indicatorPaddingPx = this._getIndicatorPaddingPx();
+    var indicatorWidthPx   = this._getIndicatorWidthPx();
+    var indicatorPosition  = this._getIndicatorPosition();
+
+    // add a pixel to account for the default input border
+    this.indicatorElem.css(indicatorPosition, indicatorPaddingPx+1);
+
+    // left-side padding
     // + indicator width
-    // + right-side spacing
-    var indicatorPaddingPx = 4 + indicatorWidthPx + 4;
-    this.elem.css({'padding-right': indicatorPaddingPx + 'px'});
+    // + right-side padding
```

Comment:
@kellyredding - really minor, but now that you can customize position I don't think this is always "right-side" padding